src/script/page/LeftSidebar/panels/Preferences.tsx

```diff
@@ -116,7 +116,7 @@ const Preferences: React.FC<PreferencesProps> = ({contentViewModel, teamReposito
 
   return (
     <ListWrapper id="preferences" header={t('preferencesHeadline')} onClose={onClose}>
-      <ul className="left-list-items no-scroll preferences-list-items">
+      <ul role="tablist" aria-label="Prefrences tabs" className="left-list-items no-scroll preferences-list-items">
```

Comment:
This probably needs a copy that will be translated for the label

Reply:
done